activerecord/lib/active_record/core.rb

```diff
@@ -87,6 +87,8 @@ def self.configurations
 
       mattr_accessor :maintain_test_schema, instance_accessor: false
 
+      mattr_accessor :require_belongs_to, instance_accessor: false
+
```

Comment:
Out of context, I wouldn't know what this "require belongs to" means. Is there a clearer config name?